packages/dashboard-message-bus/lib/utils.ts

```diff
@@ -79,3 +92,21 @@ export const sendAndAwait = (socket: WebSocket, message: Message) => {
     socket.send(encodedMessage);
   });
 };
+
+const getLocalInterfaceAddresses = () => {
+  return Object.keys(networkInterfaces()).flatMap(name =>
+    networkInterfaces()[name].map(iface => iface.address)
+  );
```

Comment:
I believe `networkInterfaces` ends up in a system call, so the results can change between calls, even within a single node.js event loop. I could be wrong, but I'd unless you want to investigate i'd just cache the return of `networkInterfaces()` and use it for the keys and within the `flatMap`. (but if you do investigate and learn anything interesting please teach me your learnings!)

Reply:
Ah good catch - will do.